.gitignore

```diff
@@ -18,4 +18,8 @@ target/
 *.iml
 *.iws
 /out
-dependency-reduced-pom.xml
+.flattened-pom.xml
```

Comment:
Is this necessary? I tried to make sure `flatten-maven-plugin` always outputs to `target`, have I missed something?

Reply:
Maybe not, I just replaced `dependency-reduced-pom.xml` with `.flattened-pom.xml`. Didn't check if this applies.